app/src/main/java/org/hisp/dhis/android/dataentry/main/home/HomeViewModel.java

```diff
@@ -7,12 +7,10 @@
 import com.gabrielittner.auto.value.cursor.ColumnName;
 import com.google.auto.value.AutoValue;
 
-import org.hisp.dhis.android.core.common.BaseIdentifiableObjectModel;
-
 @AutoValue
 abstract class HomeViewModel {
 
-    static class Columns extends BaseIdentifiableObjectModel.Columns {
+    static class Columns {
```

Comment:
Do we need Columns? What about just hardcoding column indices within conversion method?